README.md

```diff
@@ -1,36 +1,78 @@
 Piper
 =====
 
-GTK application to configure gaming mice. Piper uses libratbag via ratbagd.
+Piper is a GTK+ application to configure gaming mice, using libratbag via
+ratbagd.
+
+Piper is currently being redesigned. For the mockups, see the [Redesign
+Wiki][wiki-redesign].
 
 Dependencies
 ============
 
-Piper uses the ratbagd Python bindings. It doesn't require ratbagd itself to
-be running, but well, it'll be a little useless without ratbagd.
+* python3
+* gobject-introspection
+* gtk3
+* python-gobject
+* python-cairo
+* python-lxml
+* libratbag
```

Comment:
yeah, it's easier to just link to the top where it has all the import statements...

technically, libratbag isn't even a dependency, piper is just a tad useless without it. But you do get to see a pretty mouse trap :)